docker/README.md

```diff
@@ -9,10 +9,52 @@ You will need to install Docker to be able to run any of the following scripts.
 Make sure you have [Docker](https://docker.com) installed and configured by
 running `docker -v`.
 
-For HPI VMs, [this
-script](https://gist.github.com/johannesvedder/0fafcbeabe8e069f96085bfedaebd9d0)
+For HPI VMs,
+[this script](https://gist.github.com/johannesvedder/0fafcbeabe8e069f96085bfedaebd9d0)
```

Comment:
Why not include this in this repo?

Reply:
We probably don't want to make recommendations on how to install docker for the general majority of people running this repo outside of HPI perimeter's.